sdk/src/derivation_path.rs

```diff
@@ -123,8 +129,10 @@ impl DerivationPath {
         }
     }
 
-    // Only accepts single query string pair of type `key`
-    pub fn from_uri(uri: &URIReference<'_>) -> Result<Option<Self>, DerivationPathError> {
+    pub fn from_uri(
+        uri: &URIReference<'_>,
+        key_only: bool,
```

Comment:
I put this in to maintain current remote-wallet functionality for now. However, it should be fairly easy to support absolute paths in remote-wallet as well. This line is the only issue: https://github.com/solana-labs/solana/blob/1fe131a5bb4eca7a7a0fb83dbeaacdf1f640ca60/remote-wallet/src/ledger.rs#L378

Reply:
sgtm!  Wdyt about hiding the private bool interface behind two public wrappers for each case?